README.md

```diff
@@ -43,6 +43,14 @@ Feel free to check the [issues page](https://github.com/Gegardus/math-magicians/
 - GitHub: [@githubhandle](https://github.com/Gegardus)
 - Twitter: [@twitterhandle](https://twitter.com/Gegardus)
 
```

Comment:
- [ ] Please, kindly add the "testing" section from the [Microverse readme template](https://github.com/microverseinc/readme-template) to your project, and include instructions on how to run the tests.

That way, you will be able to showcase your hard work to potential employers and users 💪

Reply:
Done. Thank you @Meltrust.